examples/docs/es/transfer.md

```diff
@@ -106,11 +106,12 @@
 ```
 :::
 
-### Filterable
+### Filtrar
 
-You can search and filter data items.
+Puedes buscar y filtrar los items.
 
-:::demo Set the `filterable` attribute to `true` to enable filter mode. By default, if the data item `label` contains the search keyword, it will be included in the search result. Also, you can implement you own filter method with the `filter-method` attribute. It takes a method and passes search keyword and each data item to it whenever the keyword changes. For a certain data item, if the method returns true, it will be included in the result list.
+::demo Pon el atributo `filterable` a true para permitir el filtrado.
+Por defecto si el `label` del item contiene el término buscado será incluido en el resultado. También puedes implementar tu propio método de filtrado con el atributo `filter-method`, que recibe un método y le pasa la búsqueda y cada item. Los items para los que devuelva true serán incluidos en el resultado de la búsqueda.
```

Comment:
A demo's description can only have one line, otherwise the demo would be broken. Could you keep this demo's description in line 113?